.travis.yml

```diff
@@ -6,7 +6,11 @@ services:
   - postgresql
 env:
   - NODE_ENV=tests
+install:
```

Comment:
Interesting, now we're getting the following error:

> sh: 1: npm-run-all: not found

I wonder if this is causing <kbd>$ npm install</kbd> (or <kbd>$ npm ci</kbd>) to not get called, and maybe we should rename this to "before_install:", similar to https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#using-a-specific-npm-version

Reply:
Yeah, just updated `install` to do both `pip` and `npm`